build.sh

```diff
@@ -0,0 +1,8 @@
+mkdir build
+git submodule init
+git submodule update
+./vcpkg/vcpkg install nlohmann-json
+./vcpkg/vcpkg install spdlog
+./vcpkg/vcpkg install indicators
```

Comment:
This file has to leave. If you want to keep it that's fine, but it can't be in the repo